### ISSUE TITLE
[UX] Links in description of a card are not underlined

### ISSUE THREAD

**Issue by mfaure** [wekan/wekan#330]
Links in the description of a card can't be distinguished from plain text as they are not underlined.

This is a both a UX issue and an accessibility issue, as user must hover each part of the texts to find whether there is a link or not. Not really user-friendly :)

References:

* [WCAG 1.4.1](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)
* [Technique G183](http://www.w3.org/TR/WCAG-TECHS/G183.html)
* [Technique F73](http://www.w3.org/TR/WCAG-TECHS/F73.html) (this is the exact case here)
* French [RGAA 10.6.1](https://references.modernisation.gouv.fr/referentiel-technique-0#content-critre-106-a-dans-chaque-page-web-chaque-lien-dont-la-nature-nest-pas-vidente-est-il-visible-par-rapport-au-texte-environnant-)

Screenshot

![links not underlined](https://cloud.githubusercontent.com/assets/1173014/10119420/8c290b4a-6495-11e5-8a21-df7f60f7b5bb.png)

**Comment by mfaure** [comment]
**Suggested fix**
Add in the CSS:

```css
.card-details-canvas .viewer a {
    text-decoration: underline;
}
```

**Comment by mquandalle** [comment]
This was actually fixed as a side effect of 98d7278d and will be released [in the upcoming v0.10](https://github.com/wekan/wekan/issues/425).
